tests/api_connexion/endpoints/test_dag_endpoint.py

```diff
@@ -468,7 +520,7 @@ def test_should_respond_200_with_granular_dag_access(self):
     def test_should_respond_200_and_handle_pagination(self, url, expected_dag_ids):
         self._create_dag_models(10)
 
-        response = self.client.get(url, environ_overrides={'REMOTE_USER': "test"})
+        response = self.client.get(url + "&sort=asc", environ_overrides={'REMOTE_USER': "test"})
```

Comment:
This test shouldn't really need changing - if its failing without this change then you should update the endpoint code to have this as the default.

Reply:
@bbovenzi is suggesting that the default should be latest first. It's a breaking change but makes sense. Should I go ahead with updating the endpoint code?🤔